assets/css/custom.css

```diff
@@ -0,0 +1,145 @@
+.img-c {
+    width: 100%;
+    height: 18.875rem;
+    object-fit: cover;
+}
+
+.img-sm {
+    width: 100%;
+    height: 10.5rem;
+    object-fit: cover;
+}
+
+.img-xs {
+    width: 100%;
+    height: 7.5rem;
+    object-fit: cover;
+}
+
+.window-100{
+    width: 70%;
+}
+
+.window-80{
+    width: 80%;
+}
+
+.window-70{
+    width: 70%;
+}
+
+.window-60{
+    width: 60%;
+}
+
+.window-50{
+    width: 50%;
+}
+
+
+.width-100{
+    width: 70%;
+}
+
+.width-80{
+    width: 80%;
+}
+
+.width-70{
+    width: 70%;
+}
+
+.width-60{
+    width: 60%;
+}
+
+.width-50{
+    width: 50%;
+}
+
+
+.text-filter{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 1rem;
+}
+
+.mycart-amount-list{
+    display: flex;
+    flex-direction:column ;
+    gap: 2px;
+}
+.mycart-amout-desc{
+    display: flex;
+    flex-direction: row;
```

Comment:
`flex-direction: row;` is the default values for flexbox, you don't need to specify it here

Reply:
okay

Reply:
this change in cart page i will do that